.gitmodules

```diff
@@ -1,7 +1,3 @@
-[submodule "src/nacl_repy"]
```

Comment:
I'm worried about this merge making changes TO develop here. What's going on?

Reply:
By here I mean the gitmodules file in general.

Reply:
When I was looking to install lind from source, these submodules (especially nacl_repy) caused issues when building. As we don't use nacl_repy anymore, I was thinking they should also be removed from gitmodules. 
Nacl_repy is also duplicate, so is safeposix rust. ([submodule "src/--force"] refers to the same url = https://github.com/Lind-Project/safeposix-rust.git)

Reply:
Although I accept that this might not be the best branch to merge these changes.

Reply:
Removing nacl_repy is totally fine I think, lind_experiments also has not been used for a long time

Reply:
Ok this is good.